Plate_2_data/4.processing_features/scripts/2.combine_sc_runs_plate2_PBMC.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Combine runs run parquet files into one
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pandas as pd
+import pathlib
+import gc
```

Comment:
Recommend adding a short code comment on what this package does. It looks like it is very useful for memory management given how big each batch is to combine.